opm/autodiff/SimulatorFullyImplicitBlackoilOutputEbos.hpp

```diff
@@ -743,5 +745,20 @@ namespace Opm
 
         writeTimeStepWithCellProperties(timer, localState, localWellState, cellData, substep);
     }
+
+
+    bool BlackoilOutputWriterEbos::hasFIPSummaryKeywords() const
+    {
+      const SummaryConfig& summaryConfig = eclipseState_.getSummaryConfig();
+      return detail::hasFRBKeyword(summaryConfig, "WIP")  ||
```

Comment:
Seems you are not working on the latest version; I force pushed ~1 hour ago. The `required_fields` should look like this: https://github.com/OPM/opm-parser/pull/1036/files#diff-ee16a8df9fe1000dd274a1fa67e078bbR93